.github/workflows/test.yml

```diff
@@ -21,8 +21,14 @@ jobs:
     - name: Update submodule
       run: git submodule update --init --recursive
 
+    - name: Persist artifacts
+      run: ./.github/persist.sh
+      env:
+        PERSIST_DIR: /root/actions-runner/persist/
+        CCACHE_CONFIGPATH=/root/actions-runner/persist/ccache-config
```

Comment:
Check if there is a global environment for all actions, this way `CCACHE_CONFIGPATH` will have to be repeated for the "Build and test arc-mlir and arc-script" action.

The syntax looks fishy: Why is it `:` in one assignment and `=` in the other.

Reply:
This explains why the syntax highlighting didnt work 😅